compiler/backend/presLangScript.sml

```diff
@@ -33,8 +39,7 @@ val _ = Datatype`
     | Plit lit
     (* TODO: Consider doing what we did with op above for the patterns below, in
```

Comment:
I think we should drop this comment. This is exactly the TODO I fixed. Good catch.